src/content/wiki/systems/worker.mdoc

```diff
@@ -31,7 +31,7 @@ For example, Unbreaking II counts as 2 enchantments, Efficiency V counts as 5 en
 
 | Work Hut Level | Max Tool/Sword Level                                                                                                                     |
 | -------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
-| 0              | Wood or Gold + 0 enchantments                                                                                                            |
+| 0              | Stone + 0 enchantments                                                                                                            |
```

Comment:
So far I have not seen any other worker that can be a level 0, so it would be "any". ;-)

Apart from the technicality: I did not mention the (or wood/gold) part, so it would be more elegant to stick to the same format anyhow, as in "Stone + 0 enchantments (OR Wood/Gold + 1 enchantment); only applicable for builder profession"

---
Apart from that change, according to MotionlessTrain the enchantment "counting" appears to have been changed to look for the highest level present only, instead of adding up all levels of existing enchantments, so that part needs an update as well.